lib/testing_appserver/modules/documents/modules/documents_modules.rb

```diff
@@ -25,8 +27,9 @@ module DocumentsModules
     include DocumentsSelectAllFilter
     include TopToolbar
 
-    def create_file_from_action(document_type, title)
+    def create_file_from_action(document_type, title, form_template = nil)
```

Comment:
I think if there is more than two arguments - better used name arguments, so there is no mismatch in it's calls

like
```
create_file_from_action(document_type, title, form_template: nil)
```